### DISCUSSION TITLE
APOE: regression with multiple sites

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1497] [General]
error from running 3 sites across 3 machines
"Pipeline error from central node\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/remote.py\", line 260, in <module>\n sys.stdout.write(remote_0(PARAM_DICT))\n File \"/computation/scripts/remote.py\", line 41, in remote_0\n covar_keys, unique_count = return_uniques_and_counts(df)\n File \"/computation/scripts/remote_ancillary.py\", line 31, in return_uniques_and_counts\n flat_list = [item for sublist in row for item in sublist]\n File \"/computation/scripts/remote_ancillary.py\", line 31, in <listcomp>\n flat_list = [item for sublist in row for item in sublist]\nTypeError: 'float' object is not iterable\n"

**Comment by krootesmurdy** [comment]
additional error:
"Pipeline error from pipeline 6335f3207764e538a5379100 user: krootesmurdy\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/local.py\", line 189, in <module>\n sys.stdout.write(local_1(PARAM_DICT))\n File \"/computation/scripts/local.py\", line 74, in local_1\n _, local_stats_list = local_stats_to_dict_numba(args, encoded_X, y)\n File \"/computation/scripts/local_ancillary.py\", line 94, in local_stats_to_dict_numba\n params, _, tvalues, rsquared, dof_global = gather_local_stats(X1, y)\n File \"/usr/local/lib/python3.7/site-packages/numba/np/linalg.py\", line 824, in _inv_err_handler\n \"Matrix is singular to machine precision.\")\nnumpy.linalg.LinAlgError: Matrix is singular to machine precision.\n"

seemed to be solved with changing the covariate from number to string

**Reply by krootesmurdy** [reply]
same error: "Pipeline error from pipeline 6335f1597764e538a53790cd user: Emory\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/local.py\", line 189, in <module>\n sys.stdout.write(local_1(PARAM_DICT))\n File \"/computation/scripts/local.py\", line 74, in local_1\n _, local_stats_list = local_stats_to_dict_numba(args, encoded_X, y)\n File \"/computation/scripts/local_ancillary.py\", line 94, in local_stats_to_dict_numba\n params, _, tvalues, rsquared, dof_global = gather_local_stats(X1, y)\n File \"/usr/local/lib/python3.7/site-packages/numba/np/linalg.py\", line 824, in _inv_err_handler\n \"Matrix is singular to machine precision.\")\nnumpy.linalg.LinAlgError: Matrix is singular to machine precision.\n"

**Comment by pixelsaurus** [comment]
Latest error: "Pipeline error from central node\n Error details: Computation failed with exitcode 1 and stderr Traceback (most recent call last):\n File \"/computation/scripts/remote.py\", line 260, in <module>\n sys.stdout.write(remote_0(PARAM_DICT))\n File \"/computation/scripts/remote.py\", line 41, in remote_0\n covar_keys, unique_count = return_uniques_and_counts(df)\n File \"/computation/scripts/remote_ancillary.py\", line 31, in return_uniques_and_counts\n flat_list = [item for sublist in row for item in sublist]\n File \"/computation/scripts/remote_ancillary.py\", line 31, in <listcomp>\n flat_list = [item for sublist in row for item in sublist]\nTypeError: 'float' object is not iterable\n"

<img width="1085" alt="Screen Shot 2022-09-29 at 2 21 04 PM" src="https://user-images.githubusercontent.com/664834/193134075-96e72681-7bde-4389-ad93-176492358d93.png">

**Reply by rssk** [reply]
did you get to dump the specs on these?
